pages/urls/api/sequences.py

```diff
@@ -36,7 +36,7 @@
                 SequenceAPIView,
                 basename='api_sequences')
 urlpatterns = [
-    path('sequences/<slug:sequence>/<int:rank>/events/<username>/mark-attendance',
+    path('sequences/<slug:sequence>/<int:rank>/<username>/mark-attendance',
```

Comment:
It is possible we want to use a URL path that makes it easy to write permission rules (i.e. that API endpoint is for instructors. Students shouldn't be able to trigger it). Otherwise ready to merge.